template/Makefile.in

```diff
@@ -296,6 +303,16 @@ $(LIBRUBY_A):
 		@-[ -z "$(EXTSTATIC)" ] || $(PRE_LIBRUBY_UPDATE)
 		$(ECHO) linking static-library $@
 		$(Q) $(AR) $(ARFLAGS) $@ $(LIBRUBY_A_OBJS) $(INITOBJS)
+# In YJIT builds, merge libyjit.a with libruby_static.a
```

Comment:
I'm uncertain if all `make` accept a comment in middle of a recipe.

Reply:
Yeah, that was a bit risky. Addressed in f04da3cf8798e25885a2383737b1e40a6d0cbeb3